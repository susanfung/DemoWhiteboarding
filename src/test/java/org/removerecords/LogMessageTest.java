package org.removerecords;

import com.spun.util.logger.SimpleLogger;
import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

class LogMessageTest {

    @Test
    void outputMessage() {
        StringBuffer log = SimpleLogger.logToString();
        new LogMessage("Cannot connect to service", "warning").outputMessage();
        Approvals.verify(log);
    }
}