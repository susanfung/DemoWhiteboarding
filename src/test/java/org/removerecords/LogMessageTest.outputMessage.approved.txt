Event: Cannot connect to service
